<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>node_style_form</class>
 <widget class="QWidget" name="node_style_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>141</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Curve Control</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QDoubleSpinBox" name="handle_length_spinner">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string>px</string>
     </property>
     <property name="maximum">
      <double>8192.000000000000000</double>
     </property>
     <property name="value">
      <double>24.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Crossing Gap</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="crossing_gap_spinner">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string>px</string>
     </property>
     <property name="maximum">
      <double>8192.000000000000000</double>
     </property>
     <property name="value">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Min Cusp Angle</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDoubleSpinBox" name="cusp_angle_spinner">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string>Â°</string>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>15.000000000000000</double>
     </property>
     <property name="value">
      <double>225.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Cusp Distance</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QDoubleSpinBox" name="cusp_distance_spinner">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string>px</string>
     </property>
     <property name="minimum">
      <double>-8192.000000000000000</double>
     </property>
     <property name="maximum">
      <double>8129.000000000000000</double>
     </property>
     <property name="value">
      <double>32.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Curve Style</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="style_combo">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Pointed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Round</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ogee</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Polygonal</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="handle_length_check">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="crossing_gap_check">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="cusp_angle_check">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="cusp_distance_check">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="style_check">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>handle_length_spinner</sender>
   <signal>valueChanged(double)</signal>
   <receiver>node_style_form</receiver>
   <slot>anything_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>crossing_gap_spinner</sender>
   <signal>valueChanged(double)</signal>
   <receiver>node_style_form</receiver>
   <slot>anything_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cusp_angle_spinner</sender>
   <signal>valueChanged(double)</signal>
   <receiver>node_style_form</receiver>
   <slot>anything_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cusp_distance_spinner</sender>
   <signal>valueChanged(double)</signal>
   <receiver>node_style_form</receiver>
   <slot>anything_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>handle_length_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>handle_length_spinner</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>13</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>crossing_gap_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>crossing_gap_spinner</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cusp_angle_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>cusp_angle_spinner</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>13</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cusp_distance_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>cusp_distance_spinner</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>14</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>style_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>style_combo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>handle_length_check</sender>
   <signal>clicked()</signal>
   <receiver>node_style_form</receiver>
   <slot>anything_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>-32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>crossing_gap_check</sender>
   <signal>clicked()</signal>
   <receiver>node_style_form</receiver>
   <slot>anything_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>12</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cusp_angle_check</sender>
   <signal>clicked()</signal>
   <receiver>node_style_form</receiver>
   <slot>anything_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cusp_distance_check</sender>
   <signal>clicked()</signal>
   <receiver>node_style_form</receiver>
   <slot>anything_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>style_check</sender>
   <signal>clicked()</signal>
   <receiver>node_style_form</receiver>
   <slot>anything_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>anything_changed()</slot>
 </slots>
</ui>
